### PR TITLE
Added documentation for adding unsupported Tuya devices

### PR DESCRIPTION
**Pull request by drzony**
As promised in https://github.com/Koenkk/zigbee-herdsman-converters/pull/1742

### CODE REVIEW

File: docs/how_tos/how_to_support_new_tuya_devices.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+---
+# How to support new devices
```
Comment (Koenkk) [inline docs/how_tos/how_to_support_new_tuya_devices.md:3]: Should be 'How to support new TuYa devices' I think

File: docs/how_tos/how_to_support_new_devices.md [hunk 2]
```diff
@@ -28,7 +28,13 @@ The next step is the to add an entry of your device to `node_modules/zigbee-herd
 
 ```js
 {
-    zigbeeModel: ['lumi.sens'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
```
Comment (Koenkk) [inline docs/how_tos/how_to_support_new_devices.md:31]: The `zigbeeModel` is the preferred way of adding a device and is the easiest one. I think we should not mention fingerprint here but only in the how_to_support_new_tuya_devices

Comment (drzony) [reply docs/how_tos/how_to_support_new_devices.md]: Fixed

File: docs/how_tos/how_to_support_new_tuya_devices.md (hunk 1 above)
Comment (drzony) [reply docs/how_tos/how_to_support_new_tuya_devices.md]: Fixed

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**